ui/components/gridTable/gridTable.scss

```diff
@@ -33,7 +33,7 @@ $grid-column-gap: 2px;
   input[type=text] {
     text-align: left;
   }
-  input, span, select {
+  input, select {
```

Comment:
This was only used for IQ +-, so I put that specifically in the spells style.
The span was too liberal, and caused pain elsewhere.